### PR TITLE
Brig physician tweaks

### PR DESCRIPTION
**Pull request by VictorVow**
## About The Pull Request
Few small tweaks:
- An security internals box so their pepperspray can actually be safely used
- A deputy security belt for their locker rather than using taking the officer's designated one
- Their vest can hold internals tanks now
- Their locker now contains a blood filter 

## Why It's Good For The Game
QoL

## Changelog
:cl:
add: Small tweaks to brig phys locker and loadout.  
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
please dont give them security gear, physec is fucking cancer

**Comment by VictorVow** [comment]
They always take a security belt from the lockers so they're not getting anything extra here.

**Comment by Isy232** [comment]
Brig Phys is mindshielded roundstart so pretty much every antag is only gonna kill them, brigphys shouldn't be out patrolling but being able to defend themselves is good.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -80,6 +80,7 @@
 	new	/obj/item/storage/firstaid/o2(src)
 	new	/obj/item/storage/firstaid/brute(src)
 	new /obj/item/storage/belt/medical(src)
+	new /obj/item/storage/belt/security/full(src)
```
Comment (zeskorion) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: please dont give them validhunting gear. actually, it'd be a better idea to remove that fucking locker room access entirely

Comment (VictorVow) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Imagine trying to validhunt with only a baton. They have no gun, have everything else anyway, this change just means there's always enough for when all job slots are full.

Comment (zeskorion) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: a secbelt is full of gear designed for validhunting. give them deputy tools, maybe

Comment (qwertyquerty) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: I agree with zesko here

Comment (VictorVow) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Alright swapped for deputy belt

**Review by zeskorion — CHANGES_REQUESTED** [review]
remove their access to the security lockers as well, please, or they'll just grab those instead

**Comment by VictorVow** [comment]
Security lockers is the same access as various other security things including security office, gear room (for both vendors), security posts etc which they should have access to. 

Like grabbing a vest and helmet roundstart it's an administration issue and can be considered powergaming to grab all this stuff as a brig phys when they've already got their own locker.

**Comment by ghost** [comment]
> Security lockers is the same access as various other security things including security office, gear room (for both vendors), security posts etc which they should have access to.
> 
> Like grabbing a vest and helmet roundstart it's an administration issue and can be considered powergaming to grab all this stuff as a brig phys when they've already got their own locker.

Why burden the admins with it when you can mechanically restrict it?

**Comment by VictorVow** [comment]
Because it's excessive to create a new access level just for lockers and have to edit these on all maps

File: code/modules/clothing/suits/labcoat.dm [hunk 2]
```diff
@@ -22,12 +22,19 @@
...
+/obj/item/clothing/suit/toggle/labcoat/brigphys
 	name = "security medic's labcoat"
 	icon_state = "labcoat_sec"
 	item_state = "labcoat_sec"
 	armor = list(melee = 10, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 10, rad = 0, fire = 50, acid = 50)
 
+/obj/item/clothing/suit/toggle/labcoat/brigphys/vest
+	name = "brig physician's vest"
+	desc = "A lightweight vest worn by the Brig Physician."
+	icon_state = "brig_phys_vest"
```
Comment (PowerfulBacon) [inline code/modules/clothing/suits/labcoat.dm]: This icon state doesn't support toggleability

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/labcoat.dm (hunk 2 above)
Comment (VictorVow) [reply code/modules/clothing/suits/labcoat.dm]: oof, I'll revert that

Comment (VictorVow) [reply code/modules/clothing/suits/labcoat.dm]: PR should be ready now

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**